webapp_tracet/twistd_comet_wrapper.py

```diff
@@ -59,7 +59,7 @@ def output_popen_stdout(process):
 
 
     logger.info("Starting twistd command:")
-    twistd_commad = f"twistd --pidfile /tmp/twistd_comet.pid -n comet --local-ivo=ivo://hotwired.org/test {remote_command} --cmd=upload_xml.py"
+    twistd_commad = f"twistd --pidfile /tmp/twistd_comet.pid -n comet -- verbose--local-ivo=ivo://hotwired.org/test {remote_command} --cmd=upload_xml.py"
```

Comment:
Need a space between `--verbose` and `--local-ivo`. Also recommend that we use `-v -v` which is extra verbose, this is what Andrew's log have and I think it's useful.